core/src/mindustry/world/blocks/campaign/LaunchPad.java

```diff
@@ -196,8 +204,12 @@ public void draw(){
             Draw.color();
 
             Draw.z(Layer.weather - 1);
-
-            TextureRegion region = Core.atlas.find("launchpod");
+            if(podRegion.found()){
+                region = podRegion;
+            }
+            else{
+                region = podRegionDefault;
+            }
```

Comment:
might need `TextureRegion` infront of it as well since it doesn't seen to be visibly defined, awaiting ci 🤔 

```suggestion
            region = podRegion.found() ? podRegion : podRegionDefault;
```